src/sentry/backup/exports.py

```diff
@@ -197,6 +198,9 @@ def export_in_config_scope(src, *, indent: int = 2, printer=click.echo):
 
     # Pick out all users with admin privileges.
     admin_user_pks: set[int] = set()
+    admin_user_pks.update(
```

Comment:
So`UserPermission` and `UserRoleUser` doesn't apply to just admins then?

Reply:
They do. This covers the opposite case: someone has ex: `is_staff` set, but no other `UserRole` or `UserPermissions` attached. Those folks should still be exported as "admins".